src/Ema/Multi/Generic/Motley.hs

```diff
@@ -67,6 +74,34 @@ gfromMotley ::
   NS I (RCode r)
 gfromMotley = trans_NS (Proxy @Coercible) coerce
 
+type family GMotleyRouteSubRoutes (name :: SOPM.DatatypeName) (constrs :: [SOPM.ConstructorName]) (xs :: [Type]) :: [Type] where
+  GMotleyRouteSubRoutes _ _ '[] = '[]
+  GMotleyRouteSubRoutes name (c ': cs) (() ': xs) =
+    -- TODO: The .html suffix part should be overridable.
+    SingletonRoute (Constructor2RoutePath name c ".html")
```

Comment:
This will be important because sometimes users want to generate non-HTML files, like RSS. `data R = R_Blog ... | R_Feed` where the last constructor should be `feed.xml`.